mission_control/README.md

```diff
@@ -0,0 +1,22 @@
+# Mission Control
+The mission control package provides an implementation of missions using behavior trees
+It is compliant with the the [BehaviorTree.CPP](https://github.com/BehaviorTree/BehaviorTree.CPP) library.  
+
+## Features
+
+- The mission control implements the interface with the Jaus Ros Bridge ROS Package.
+
+The behaviors/actions implemented were:
```

Comment:
@patapufate how about 
```suggestion
- Actions supported are:
```
and then indent the list below.

Reply:
Done